BackEndApp/BackEndApp/settings.py

```diff
@@ -125,3 +126,38 @@
 EMAIL_HOST_USER = os.environ.get("EMAIL_HOST_USER")
 DEFAULT_FROM_EMAIL = os.environ.get("EMAIL_HOST_USER")
 EMAIL_HOST_PASSWORD = os.environ.get("EMAIL_HOST_PASSWORD")
+
+logging.config.dictConfig({
+    'version': 1,
+    'disable_existing_loggers': False,
+    'formatters': {
+        'colored': {
+			'()': 'colorlog.ColoredFormatter',
+			'format': "%(log_color)s %(asctime)s %(name)-12s %(levelname)-8s %(message)s"
+        },
```

Comment:
Instead of having a separate formatter `colored` just add it to console so it's the default

Reply:
done, please have a look.

Reply:
Nice :v: